mapping:
  - 'local'
  - 'rpi8'
  - 'rpi7'
  - 'rpi6'
  - 'rpi5'
  - 'rpi3'

adjacency:
  - [0, 1, 0, 0, 0, 0]
  - [1, 0, 1, 0, 0, 0]
  - [0, 1, 0, 1, 0, 0]
  - [0, 0, 1, 0, 1, 0]
  - [0, 0, 0, 1, 0, 1]
  - [0, 0, 0, 0, 1, 0]

initial_values:
  range: [0, 360, 1]  # star, stop, step // if step == 0, range is continuous

alpha: 0.15
